src/System.Collections/src/Configurations.props

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <!-- Configurations not yet building netcoreapp1.2corert; -->
```

Comment:
I do not think you need to bother commenting on netcoreapp1.2corert config missing. It is heavily bit-rotten and it would be best to redo it from scratch.